### PR TITLE
Init example

### PR DESCRIPTION
**Pull request by banadiga**

### CODE REVIEW

File: hapi-example/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  ],
+  "author": "Ihor Banadiga",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/LvivNodeJsClub/frameworks-for-REST-API-in-Node.js/issues"
+  },
+  "homepage": "https://github.com/LvivNodeJsClub/frameworks-for-REST-API-in-Node.js#readme",
+  "dependencies": {
+    "@hapi/hapi": "^18.3.2",
+    "hapi": "^18.1.0"
```
Comment (tmorozov) [inline hapi-example/package.json:25]: why do we need 2 modules hapi & @hapi/hapi?

File: loopback-example/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (tmorozov) [inline loopback-example/.vscode/settings.json:1]: do we need to commit .vscode ?

**Review by tmorozov — APPROVED** [review]

File: hapi-example/package.json (hunk 1 above)
Comment (banadiga) [reply hapi-example/package.json:25]: will be fixed in new PR

File: loopback-example/.vscode/settings.json (hunk 2 above)
Comment (banadiga) [reply loopback-example/.vscode/settings.json:1]: it was auto-generated by `lb4` and added as an example

**Pull request merged**